### PR TITLE
CSC Marketing Updates

### PR DESCRIPTION
**Pull request by hannahbergam**
This covers the CSC page updates from this figma: https://www.figma.com/file/xPFXkCz4m96FKP3RJWQ1i4/Curriculum-Launch---July-2023?type=design&node-id=901-84&mode=design&t=0HMcuo4M3GwFgDEf-0

The sections  in the pic below contain the majority of the changes! The rest are copy changes.

Before:
Screenshot 2024-02-15 at 3 27 24 PM

After:
Screenshot 2024-02-15 at 3 08 35 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1322

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by TurnerRiley — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/educate/csc/index.haml [hunk 1]
```diff
@@ -50,74 +50,46 @@ theme: responsive_full_width
...
+    %h2=hoc_s(:csc_hour_long_lessons)
+    %p=hoc_s(:csc_hour_long_lessons_desc)
+    .action-block__wrapper.action-block__wrapper--two-col
       .action-block.action-block--one-col.flex-space-between
         .content-wrapper
           %p.overline=hoc_s(:module_subject_english_language_arts)
-          %h3=hoc_s(:module_name_coding_w_poetry)
-          %img{src: "/images/csc-module-coding-w-poetry.png", alt: ""}
+          %h3=hoc_s(:tutorial_poemart_name)
+          %img{src: "/images/csc-module-poem-art.png", alt: "", style: "height: 100%, max-width: 430px"}
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/csc/index.haml]: These inline styles can be removed: `, style: "height: 100%, max-width: 430px"`
```suggestion
          %img{src: "/images/csc-module-poem-art.png", alt: ""}
```

File: pegasus/sites.v3/code.org/public/educate/csc/index.haml [hunk 2]
```diff
@@ -50,74 +50,46 @@ theme: responsive_full_width
...
+      %a.link-button{href: "#csc-hour-long-lessons"}
+        =hoc_s(:csc_hour_long_lessons)
+
+%section#csc-week-long-modules
+  .wrapper
+    %h2=hoc_s(:csc_week_long_modules)
     %p=hoc_s(:csc_page_find_a_module_desc)
-    .action-block__wrapper.action-block__wrapper--three-col
+    = view :carousel_csc_week_long
+
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/csc/index.haml:70]: Let's add a `= view :section_divider_line` here between these two sections so it breaks them up a bit.

**Review by kelbyhawn — COMMENTED** [review]
Couple suggestions, but otherwise looks great! 🙌

File: pegasus/sites.v3/code.org/public/educate/csc/index.haml (hunk 2 above)
Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/educate/csc/index.haml:70]: Screenshot 2024-02-16 at 9 17 45 AM

Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/educate/csc/index.haml:70]: Done! Good call:)

File: pegasus/sites.v3/code.org/public/educate/csc/index.haml (hunk 1 above)
Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/educate/csc/index.haml]: Ahh good catch. I eventually just cropped the image but forgot to remove this! Marking resolved now

**Review by kelbyhawn — APPROVED** [review]
Thanks for the updates! 🎉

**Pull request merged**